sdk/python/foundation-models/system/inference/text-to-image/safe-text-to-image-online-deployment.ipynb

```diff
@@ -561,7 +563,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "> The `parameters` are optional inputs that support `num_inference_steps` and `guidance_scale`. If you need support for new parameters, please file a support ticket."
+    "> The `parameters` are optional inputs that support `num_inference_steps` and `guidance_scale`. If you need support for new parameters, please file a support ticket.\n",
+    "\n",
+    "For model `stabilityai-stable-diffusion-xl-base-1-0` need to remove parameters from inputs."
```

Comment:
"For model `stabilityai-stable-diffusion-xl-base-1-0` need to remove parameters from inputs." -> "For model `stabilityai-stable-diffusion-xl-base-1-0`, the input `parameters` is not supported."